### PR TITLE
Jarv/add hostname to common

### PR DESCRIPTION
**Pull request by jarv**
@feanil 

This PR does the following:
- Moves the logic to change the hostname into the common role, by default it won't do anything to change the hostname unless COMMON_HOSTNAME is set.
- Creates /etc/skel/.bashrc that adds an optional prefix to PS1

### CODE REVIEW

File: playbooks/roles/gh_users/tasks/main.yml [hunk 1]
```diff
@@ -12,9 +12,10 @@
 #      - mark
 
 
-- name: gh_users | checking for gh_users list
```
Comment (feanil) [inline playbooks/roles/gh_users/tasks/main.yml:15]: Why did you remove this check and instantiate the gh_users to empty?

Comment (jarv) [inline playbooks/roles/gh_users/tasks/main.yml:15]: I want to include the gh_users role to update the .skel directory with the default .bashrc (and optionally not create an users).

Comment (feanil) [inline playbooks/roles/gh_users/tasks/main.yml:15]: I'd rather see the .bashrc settings in common I think.

Comment (feanil) [inline playbooks/roles/gh_users/tasks/main.yml:15]: talked in person, this makes sense because of how the .skel directory is tied to user creation.

**Comment by feanil** [comment]
looks good.
